MafiaBot/MafiaContext.cs

```diff
@@ -16,8 +16,11 @@ namespace MafiaBot {
     public class MafiaContext : MafiaPlayers {
         private const int DiscussionTime = 30000;
         private const long CitizenVoteTime = 90000;
+        private static readonly string[] DeathMessages = File.ReadAllLines(path: "Lines/messages.txt");
```

Comment:
Why is this in the middle of the different times? It looks kinda sketch but its okay.